cvat-ui/src/components/labels-editor/labels-editor.tsx

```diff
@@ -319,6 +320,7 @@ export default class LabelsEditor
                         constructorMode === ConstructorMode.CREATE
                             && (
                                 <ConstructorCreator
+                                    labelNames={labels.map((l) => l.name)}
```

Comment:
``ConstructorCreator`` will be rerendered every time when ``LabelsEditor`` rerenders, but it isn't really needed

Reply:
@bsekachev, why not? For example when we go to another label with continue button we want to filter label with all label names.

Reply:
Sure. I mean a bit another. The component ``LabelsEditor`` has other props and states (and they may be more in the future), and when they are changed, the component ``ConstructorCreator`` also will be updated (even if it doesn't depend on these props and states), just because render method works for ``LabelsEditor`` and for each render ``ConstructorCreator`` gets new object ``labelNames``. In these case good practice is to push mapped ``labelNames`` to state and use lifecycle method ``getDerivedStateFromProps``. Another solution is to memoize ``ConstructorCreator`` and write custom comparison function that checks props better, than in default way.